app/lib/benchmarks_fixture.rb

```diff
@@ -11,14 +11,17 @@ def capacities
   end
 
   def capacity_text(capacity_id)
+    capacity_id = String(capacity_id)
```

Comment:
use a different name for `capacity_id` to illustrate how it is different from the `capacity_id` that's passed in.